README.md

```diff
@@ -85,6 +85,10 @@ Synthesizing...
 Clone output to cmdExampleAudio.wav
 ```
 
+### AutoTranslate script
+
+To use the AutoTranslate script on Windows, drag and drop an audio file onto the script or place a shortcut to the script in ```%AppData%\Microsoft\Windows\SendTo\``` and use the "Send To" context menu function on an audio file to be translated. In both cases a new .wav file with the orginal filename followed by "Translated" will be placed in the same folder. For other platforms, the same CLI flags should be used but details on context menu integration will vary by what packages are installed. 
```

Comment:
I think this is a great idea! I like that the shortcut to the .bat file will allow this to work anywhere!

I just have a few things:
- Emphasize **shortcut** to the .bat file! I think that will save some confusion for people.
- You might add that files will be _auto translated_ from a supported language to English.

Reply:
I agree with both of those and did them.